src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -62,13 +62,21 @@ export const SearchResults = ({
     isOpen: false,
     pdfSelected: '',
     pdfUrl: '',
+    pdfLabel: '',
+    cookieExistence: false,
   });
   useEffect(() => {
     const justRefreshed = prevProps?.fetching && !fetching;
     if (justRefreshed) {
       focusElement('[data-forms-focus]');
     }
   });
+  useEffect(() => {
+    const doesCookieExist = getCookie();
```

Comment:
is the return result from `getCookie` `cookie`? If so might be better calling `doesCookieExist` `cookie` instead so devs know they can expect it for cookie details

Reply:
It actually returns a boolean. I perhaps should rename it? 

![Screen Shot 2021-11-10 at 2 46 05 PM](https://user-images.githubusercontent.com/26075258/141182637-d708d638-e9ff-4516-9b0d-b1652456a428.png)

Reply:
ya `getCookie` sounds like it will return the cookie, I think if `getCookie` doesn't return back the string and a bool instead then perhaps rename it to `doesCookieExist()`?